packages/arb-token-bridge-ui/tests/support/common.ts

```diff
@@ -256,13 +256,12 @@ export async function generateActivityOnChains({
 }
 
 export async function checkForAssertions({
-  parentChainProvider
+  parentProvider,
```

Comment:
I would keep the old name, its consistent with rest of the codebase

Reply:
As discussed, this E2E is following `parentProvider` convention